patches/@ua+react-native-airship+17.2.1.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@ua/react-native-airship/src/types.ts b/node_modules/@ua/react-native-airship/src/types.ts
+index 1514436..cec598f 100644
+--- a/node_modules/@ua/react-native-airship/src/types.ts
++++ b/node_modules/@ua/react-native-airship/src/types.ts
```

Comment:
This should be applied to `node_modules/@ua/react-native-airship/lib/typescript/types.d.ts`